lib/charms/operator_libs_linux/v0/juju_systemd_notices.py

```diff
@@ -363,30 +401,13 @@ async def _async_load_services() -> None:
     that should be watched. Upon finding a service hook it's current ActiveState
     will be queried from systemd to determine it's initial state.
     """
-    global _juju_unit
-    hooks_dir = Path.cwd() / "hooks"
-    _logger.info("Loading services from hooks in %s", hooks_dir)
-
-    if not hooks_dir.exists():
-        _logger.warning("Hooks dir %s does not exist.", hooks_dir)
-        return
-
-    watched_services = []
-    # Get service-{service}-(started|stopped) hooks defined by the charm.
-    for hook in hooks_dir.iterdir():
-        match = _service_hook_regex_filter.match(hook.name)
-        if match:
-            watched_services.append(match.group("service"))
-
-    _logger.info("Services from hooks are %s", watched_services)
-    if not watched_services:
-        return
-
     bus = await MessageBus(bus_type=BusType.SYSTEM).connect()
 
     # Loop through all the services and be sure that a new watcher is
     # started for new ones.
-    for service in watched_services:
+    global _observed_services
```

Comment:
Again, no `global` required; you're just referencing it, not reassigning it.

Reply:
Dropped global reference in https://github.com/canonical/operator-libs-linux/pull/128/commits/e3ab7193bffcdc5b04efb6fcd37f4b6e960ea7fb